### DISCUSSION TITLE
File Space - Instance Count OK

### DISCUSSION THREAD

**Discussion by AGlennDa** [trimble-oss/dba-dash#1097] [Q&A] [answered]
I get a little confused when looking at the Summary, in regards to the "File Space". 
In my test scenario, I have added 9 instances in total. 
8 of them is shown in the "Instance Count N/A", and 1 of them is in the "Instance Count OK". 
I have all the same setup (collections) on all 9 instances. 

When I click on the 8 instances which is greyed out, and I change the filter to show the ones with status OK, I get to see that they are all indeed OK. So why are those instances shown as "N/A", when they are indeed available, and OK? :) 

![image](https://github.com/user-attachments/assets/9247970a-beca-40eb-96e9-cccd748b6721)

**Comment by DavidWiseman** [comment, marked as answer]
The overall server status is NA when all the files are excluded.  

If you check the Files tab for the instance, you will see that all of the database files are excluded for some reason.  Depending on the configuration it might be setup to alert when free space is low only for files without autogrow enabled.  In this case the excluded reason is "Autogrow enabled".  If you prefer to pre-grow database files manually and use autogrow as a backup, you can have it alert regardless of the autogrow setting (Uncheck *Zero autogrowth only* in the threshold config.
There might be other reasons files are excluded - if the database is a log shipped database etc.
